test/cli/CLITest.ts

```diff
@@ -5,7 +5,7 @@ import { basename, dirname, join } from 'path';
 import { sync as rimraf } from 'rimraf';
 
 function plugin(name: string): string {
-  return join(__dirname, `fixtures/plugin/${name}.js`);
+  return join(__dirname, `../fixtures/plugin/${name}.js`);
```

Comment:
Most of these sort of changes were just moving CLI tests into `test/cli` and everything else into `test/unit` to enable running the unit tests by themselves more quickly. That’s somewhat nullified by the required `tsc -p .` step, but such is life. I’m open to suggestions for ways to better speed up the tests (@mcMickJuice mentioned Jest).